### PR TITLE
Meta: fix ecma text copyright link

### PR DESCRIPTION
**Pull request by ctcpip**
old link is dead.  also eliminated the scare quotes

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 This repository is licensed according to Ecma International TC39's [Intellectual Property Policy](https://github.com/tc39/how-we-work/blob/HEAD/ip.md). In particular:
-- Natural language text is licensed under the "Alternative copyright notice" of the [Ecma text copyright policy](https://www.ecma-international.org/memento/Ecma%20copyright%20faq.htm).
+- Natural language text is licensed under the [Alternative copyright notice of the Ecma text copyright policy](https://ecma-international.org/policies/by-ipr/ecma-text-copyright-policy/#:~:text=ALTERNATIVE%20COPYRIGHT%20NOTICE%20AND%20COPYRIGHT%20LICENSE).
```
Comment (ljharb) [inline LICENSE.md:2]: we shouldn't link to something that relies on the JS-based `:~:text` thing; i've just emailed ecma to fix the link (cool URLs don't change), and to add a proper ID.

Comment (ctcpip) [reply LICENSE.md:2]: https://wicg.github.io/scroll-to-text-fragment/

Comment (ljharb) [reply LICENSE.md:2]: woof, didn't realize this was a thing. i'd def still prefer an ID tho, regardless.

Comment (ljharb) [reply LICENSE.md:2]: k, redirect is in place, and new link is:
```suggestion
- Natural language text is licensed under the [Alternative copyright notice of the Ecma text copyright policy](https://ecma-international.org/policies/by-ipr/ecma-text-copyright-policy/#alternative-copyright-notice).
```

**Comment by ctcpip** [comment]
additional linebreak to appease the `markdownlint` deities

**Review by michaelficarra — APPROVED** [review]

**Review by bakkot — APPROVED** [review]

**Review by syg — APPROVED** [review]

**Pull request merged**